hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/JetBootstrap.java

```diff
@@ -124,6 +131,11 @@ public static synchronized void executeJar(@Nonnull Supplier<JetInstance> suppli
             String[] jobArgs = args.toArray(new String[0]);
             // upcast args to Object so it's passed as a single array-typed argument
             main.invoke(null, (Object) jobArgs);
+            List<Job> jobsAfterSubmit = JetBootstrap.supplier.get().getJobs();
+            printJobStatusInfo(jobsAfterSubmit
+                    .stream()
+                    .filter(job -> !jobsBeforeSubmit.contains(job))
```

Comment:
Here, I couldn't manage how to filter jobs submitted from a different client in a proper way.